HACKING.adoc

```diff
@@ -206,6 +206,23 @@ has excellent documentation.
 
 == Development tips and tricks
 
+=== Keep merge commits when merging and cherry-picking Github PRs
+
+Having the Github PR number show up in the git log is very useful for
+later triaging. Please use either the "Merge" or "Squash and merge"
+options of Github, which produce an explicit merge-style commit with
+the PR number, and avoid "Rebase and merge", which does not.
+
```

Comment:
In light of the recent removal of "Rebase and merge", you can rephrase:

the PR number.  We recently removed the "Rebase and merge" option, precisely because it does not preserve the PR number.

Reply:
Thanks! Rephrased.